loadtest/control/simulcontroller/controller.go

```diff
@@ -302,3 +363,42 @@ func (c *SimulController) sendFailStatus(reason string) {
 func (c *SimulController) sendStopStatus() {
 	c.status <- control.UserStatus{ControllerId: c.id, User: c.user, Info: "user stopped", Code: control.USER_STATUS_STOPPED}
 }
+
+// InjectAction allows a named UserAction to be injected that is run once, at the next
+// available opportunity. These actions can be injected via the coordinator via
+// CLI or Rest API.
+func (c *SimulController) InjectAction(actionID string) control.UserActionResponse {
+	var action userAction
+	var ok bool
+
+	// include some actions that are not normally supported by SimulController
+	switch actionID {
+	case "Reload":
+		action = userAction{
+			name: "Reload",
+			run:  func(_ user.User) control.UserActionResponse { return c.reload(false) },
+		}
+	default:
+		action, ok = c.actionMap[actionID]
```

Comment:
Should we make sure this access is not concurrent with the initialization in `Run()`. They technically happen in separate goroutines so although unlikely it could be racy.

Reply:
I moved the initialization of the actions list and map to the constructor.  They are only read after New.